service/ca/pki.go

```diff
@@ -3,14 +3,39 @@ package ca
 import (
 	"fmt"
 	"strings"
+	"time"
 
+	"github.com/cenkalti/backoff"
 	"github.com/giantswarm/certctl/service/pki"
 	"github.com/giantswarm/certctl/service/token"
 	"github.com/giantswarm/certificatetpr"
 	microerror "github.com/giantswarm/microkit/error"
 )
 
-// SetupPKI creates a PKI backend and policy if one does not exists for the cluster.
+const (
+	setupPKIMaxElapsedTime = 30 * time.Second
```

Comment:
The timeouts are short because the timeout for the secrets watch is only 90 seconds. The operator needs to issue all certs in that time.